lib/client.js

```diff
@@ -557,6 +557,9 @@ export default class Client {
         options = { url: options }
       }
 
+      const appId = this._metadata && this._metadata.appId
```

Comment:
nit: consider `options.headers = addUserAgent(options.headers, this._metadata?.appId)`

Reply:
Likewise, node 12 does not support optional chaining 😢